readthedocs/projects/tasks/builds.py

```diff
@@ -700,6 +700,11 @@ def after_return(self, status, retval, task_id, args, kwargs, einfo):
         if self.data.version:
             clean_build(self.data.version)
 
+        try:
+            self.data.api_client.revoke.post()
+        except Exception:
+            log.exception("Failed to revoke build api key.", exc_info=True)
```

Comment:
I would make this just an INFO. There is no need to log this as an exception. The key will expire anyways in 3 hours.